laghos_rom.cpp

```diff
@@ -8,6 +8,7 @@ using namespace std;
 
 void ROM_Sampler::SampleSolution(const double t, const double dt, Vector const& S)
 {
+    bool addSample, addSampleF;
```

Comment:
Just a minor suggestion for better style, these `bool` variables can have more limited scope by declaring them where used. This may make the code a little clearer and avoid misuse of them.